helpdesk_mgmt_stage_server_action/tests/test_helpdesk_ticket_stage.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import SavepointCase
+
+
+class HelpdeskTicketStageServerAction(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(HelpdeskTicketStageServerAction, cls).setUpClass()
+        cls.ServerAction = cls.env["ir.actions.server"]
+        cls.HelpdeskTicket = cls.env["helpdesk.ticket"]
+        cls.HelpdeskTicketStage = cls.env["helpdesk.ticket.stage"]
+        cls.field = cls.env["ir.model.fields"]._get(cls.HelpdeskTicket._name, "user_id")
+        cls.server_action_helpdesk_ticket = cls.ServerAction.create(
+            {
+                "name": "Helpdesk Ticket Server Action",
+                "model_id": cls.env.ref("helpdesk_mgmt.model_helpdesk_ticket").id,
+                "state": "object_write",
+                "fields_lines": [
+                    (
+                        0,
+                        0,
+                        {
+                            "col1": cls.field.id,
+                            "evaluation_type": "value",
+                            "value": cls.env.user.id,
+                        },
+                    )
+                ],
+            }
+        )
+        cls.helpdesk_ticket_stage_1 = cls.HelpdeskTicketStage.create(
+            {"name": "Stage 1", "sequence": 1}
+        )
+        cls.helpdesk_ticket_stage_2 = cls.HelpdeskTicketStage.create(
+            {
+                "name": "Stage 2",
+                "action_id": cls.server_action_helpdesk_ticket.id,
+                "sequence": 2,
+            }
+        )
+
+    def test_helpdesk_ticket(self):
```

Comment:
Please complete unit tests with different cases we can have (list non exhaustive):
* action on stage when creating the ticket
* action with stage when write ticket, but with current stage (no real stage change)
* ...

And please also add test without action to check no action has been executed.